index.bs

```diff
@@ -5916,6 +5917,56 @@ When [=matching capabilities=], the extension-specific steps to match `"webauthn
         the match is unsuccessful.
     2. Otherwise, the match is successful.
 
+### <dfn>Authenticator Extension Capabilities</dfn> ### {#sctn-authenticator-extension-capabilities}
+
+Additionally, [=extension capabilities=] are defined for every [=authenticator extension=] defined in this specification:
```

Comment:
This "every" is no longer accurate after the latest update. :)

Reply:
I think it's still true.

> An extension can also be an *authenticator extension*, meaning that the extension involves communication with and processing by the authenticator.

The extensions I left here are precisely those that involve processing by the authenticator, i.e. those for which "Authenticator extension processing" is not empty.

--

I agree this text can be confusing so I added a clarification.

Reply:
Oh right, sorry, this was indeed accurate. So I don't think the clarification is really needed, but I don't mind keeping it in either.